otter/test/test_metrics.py

```diff
@@ -222,7 +221,8 @@ def test_added(self):
                 ServiceType.CLOUD_METRICS_INGEST, "POST", "ingest",
                 data=req_data, log=log).intent, noop)
         ]
-        eff = add_to_cloud_metrics(m['ttlInSeconds'], 'ord', metrics, 3, log)
+        eff = add_to_cloud_metrics(m['ttlInSeconds'], 'ord', metrics, 3,
```

Comment:
`3` feels very magical. Had to look at the docstring for `add_to_cloud_metrics` to know that `3` == `no_tenants=3` == `number of tenants = 3`. I'd prefer this to be a kwarg.